src/presentation/http/router/noteSettings.ts

```diff
@@ -87,6 +104,13 @@ const NoteSettingsRouter: FastifyPluginCallback<NoteSettingsRouterOptions> = (fa
         'authRequired',
```

Comment:
we also need policies for operations (patch, get team)  with note settings. And test cases for them